content/chapters/compute/lab/README.md

```diff
@@ -282,6 +282,201 @@ Therefore, there is also a gray area where there is no clear right answer to thi
 One such gray area are browsers.
 Some open a new tab in a separate process, while others do it in a separate thread instead.
 
+## Usage of processes and threads in apache2
+
+We'll take a look at how a real-world application - the apache2 http server - makes use of processes and threads.
+Since the server must be able to handle multiple clients at the same time, it must therefore use some form of concurrency: when a new client arrives, the server offloads the work of interacting with that client to another process or thread.
```

Comment:
```suggestion
Since the server must be able to handle multiple clients at the same time, it must therefore use some form of concurrency.
When a new client arrives, the server offloads the work of interacting with that client to another process or thread.
```